packages/webdriverio/src/utils/interception/webdriver.ts

```diff
@@ -19,6 +19,13 @@ export default class WebDriverInterception extends Interception {
         this.mockId = mockId
     }
 
+    on() {
+        throw new Error('Not implemented')
+
+        // eslint-disable-next-line no-unreachable
+        return this
```

Comment:
Had to add this, because
`typescript`: "method `on` have to return `this`"
`eslint`: "unreachable code"